shell/plugins/dashboard-store/actions.js

```diff
@@ -143,6 +143,21 @@ export default {
     }
   },
 
+  async findPageOrAll(ctx, {
+    type, context, paginatedOpt, findAllOpt
```

Comment:
not that findAllOpts isn't supplied where this is used.

however we should be safe just chopping out the pagination object and using the paginatedOpts (though it should be harmless)

Reply:
I'm not sure I'm understanding the ask here. 

Are you just wanting me to remove the object de-structuring?